src/python/evaluation/xlsx_run_tool.py

```diff
@@ -0,0 +1,170 @@
+                             'Your XLSX-file must include column-names: '
+                             f'"{ColumnName.CODE.value}" and '
+                             f'"{ColumnName.LANG.value}". Acceptable values for '
+                             f'"{ColumnName.LANG.value}" column are: '
+                             f'{LanguageVersion.PYTHON_3.value}, {LanguageVersion.JAVA_8.value}, '
+                             f'{LanguageVersion.JAVA_11.value}, {LanguageVersion.KOTLIN.value}.')
+
+    parser.add_argument('-tp', '--tool-path',
+                        default=Path('src/python/review/run_tool.py').absolute(),
+                        type=lambda value: Path(value).absolute(),
+                        help='Path to script to run on files.')
+
+    parser.add_argument('-tr', '--traceback',
+                        help='If True, column with the full inspector feedback will be added '
+                             'to the output file with results.',
+                        action='store_true')
+
+    parser.add_argument('-ofp', '--output-folder-path',
+                        help='An absolute path to the folder where file with evaluation results'
+                             'will be stored.'
+                             'Default is the path to a directory, where is the folder with xlsx_file.',
+                        # if None default path will be specified based on xlsx_file_path.
+                        default=None,
+                        type=str)
+
+    parser.add_argument('-ofn', '--output-file-name',
+                        help='Filename for that will be created to store inspection results.'
+                             f'Default is "{EvaluationArgument.RESULT_FILE_NAME_EXT.value}"',
+                        default=f'{EvaluationArgument.RESULT_FILE_NAME_EXT.value}',
+                        type=str)
+
+    parser.add_argument(run_tool_arguments.FORMAT.value.short_name,
+                        run_tool_arguments.FORMAT.value.long_name,
+                        default=OutputFormat.JSON.value,
+                        choices=OutputFormat.values(),
+                        type=str,
+                        help=f'{run_tool_arguments.FORMAT.value.description}'
+                             f'Use this argument when {EvaluationArgument.TRACEBACK.value} argument'
+                             'is enabled argument will not be used otherwise.')
+
+
+def create_dataframe(config) -> Union[int, pd.DataFrame]:
+    report = pd.DataFrame(
+        {
+            ColumnName.LANGUAGE.value: [],
+            ColumnName.CODE.value: [],
+            ColumnName.GRADE.value: [],
+        },
+    )
+
+    if config.traceback:
+        report[EvaluationArgument.TRACEBACK.value] = []
+
+    try:
+        lang_code_dataframe = pd.read_excel(config.xlsx_file_path)
+
+    except FileNotFoundError:
+        logger.error('XLSX-file with the specified name does not exists.')
+        raise FileNotFoundError
```

Comment:
It is a bad practice to raise an error without a message. You should use the following syntaxis:
```
except FileNotFoundError as e:
        logger.error('XLSX-file with the specified name does not exists.')
        raise e
```
in this case the old message will be passed/ Fix it everywhere, please